### DISCUSSION TITLE
3d support for flame

### DISCUSSION THREAD

**Discussion by cmedamine** [flame-engine/flame#2346] [General]
are you going to support 3d for flame since flutter I think they will support 3d using an impeller? thanks

**Comment by spydon** [comment]
Impeller is far away from stable on all platforms (no platforms so far) yet, but when it is, we will definitely work for more 3D support. Apart from that we would need some more shader support and access to a Z-buffer to make a good 3D game engine too.

**Reply by cmedamine** [reply]
Awesome thanks!
